src/core/ircnetwork.cpp

```diff
@@ -191,7 +191,11 @@ IrcNetworkPrivate::IrcNetworkPrivate() :
 static QHash<QString, int> numericValues(const QString& parameter)
```

Comment:
We could define `Qt::SkipEmptyParts` ourselves somewhere here above where it's used in `ircnetwork.cpp`:

```
#if QT_VERSION < QT_VERSION_CHECK(5, 14, 0)
namespace Qt {
    using SkipEmptyParts = QString::SkipEmptyParts;
}
#endif
```

This way one version check would be enough and we could unconditionally use `Qt::SkipEmptyParts` below. Furthermore, we could just throw away the version check in the future when IRC has taken over the world and Qt 5 is no longer relevant. :)

Reply:
Does that work - assigning one type of flag to another?

I tried something fancy in (I think it was the `ircmessage_p.h` header file) to try and define a `enum ircmessage::SkipBehavourFlag`  that would have the same values as the `Qt` or `QString` ones and be initialised from the version appropriate one - but the Qt libs got all nasty about trying to convert from one sort of enum to another... which was a pity as it would have been nice to replace the Qt version dependent uses of the `enum`s with a uniform `ircmessage::SkipEmptyParts` and allow that to be configured behinds the scenes during compilation with the relevant Qt libraries. 🤷‍♂️

Reply:
Feel free to push additional commits as PRs onto this PR...